language-server/modules/langserver-core/src/test/resources/codeaction/import-module/config/importModule1.json

```diff
@@ -1,32 +0,0 @@
-{
-    "line": 1,
-    "character": 5,
-    "expected": [
-        {
-            "title": "Import module 'ballerina/module1'",
-            "kind": "quickfix",
-            "diagnostics": [],
-            "edit": {
-                "documentChanges": [
-                    {
-                        "edits": [
-                          {
-                            "range": {
-                              "start": {
-                                "line": 0,
-                                "character": 0
-                              },
-                              "end": {
-                                "line": 0,
-                                "character": 0
-                              }
-                            },
-                            "newText": "import ballerina/module1;\n"
-                          }
-                        ]
-                    }
-                ]
-            }
-        }
-    ]
-}
```

Comment:
This shouldn't fail. Maybe we need to update the test case

Reply:
This test case is not failing `importModule1.json` is moved to `importModule1a.json`